templates/contacts/contactimport_create.haml

```diff
@@ -93,6 +99,19 @@
           {{ form.non_field_errors }}
 
       -block fields
+        -if can_validate_upload
+          %label#validate-upload-label(for="validate-upload-input")
+            %input#validate-upload-input(type="checkbox" name="validate_carrier")
```

Comment:
@musamusa, try to use `temba-checkbox` component. We have connected a specific library with wrappers for all basic components. When you have free time you can take a look at it https://github.com/greatnonprofits-nfp/temba-components/tree/current-ccl-version.

Reply:
Yup, I did use it at first, it was limited in feature, I'll try to remember why I removed it and post it here.

Reply:
Ok, I remember it has something to do with access to styling the label, with issue access nested shadow roots with CSS. I replaced it though, so we can sort styling some how